src/scss/modules/list-pages.scss

```diff
@@ -15,18 +15,24 @@
 }
 
 .list-page__title {
```

Comment:
Just noticed this.

Adding `align-items: center` here messes with title alignments on other pages where `list-page__title` is used.

For example:

<img width="398" alt="Screen Shot 2022-10-11 at 2 17 36 PM" src="https://user-images.githubusercontent.com/35355575/195179771-990dbcc9-12ef-4f52-96be-29d8dedb43f1.png">

Reply:
Fixed this in the latest pushed changes.